libc-top-half/musl/src/env/__init_tls.c

```diff
@@ -16,6 +16,36 @@
 volatile int __thread_list_lock;
 
 #ifndef __wasilibc_unmodified_upstream
+
+extern unsigned char __heap_base;
+extern unsigned char __data_end;
+extern unsigned char __global_base;
+extern weak unsigned char __stack_high;
+extern weak unsigned char __stack_low;
+
+static inline void setup_default_stack_size()
+{
+	ptrdiff_t stack_size;
+
+	if (&__stack_high)
+		stack_size = &__stack_high - &__stack_low;
+	else
```

Comment:
IIIUC the coding style of musl puts opening braces for (non-function) blocks on the same line as the keyword.